core/llm/index.ts

```
@@ -241,6 +241,23 @@ ${prompt}`;
 
         if (!resp.ok) {
           let text = await resp.text();
+
+          if (resp.status === 401) {
+            // supress "Unathorized error" for v0 launch, redo later for PearAI.
+            if (
+              text.includes("Please upgrade to the latest version of Continue")
+            ) {
+              console.warn(
+                `supressed error: HTTP ${resp.status} ${resp.statusText} from ${resp.url}\n\n${text}`,
+              );
+              throw {
+                status: resp.status,
+                statusText: "Unauthorized Error",
```

I wonder if there are other 401 unauth errors that we need to keep. maybe make this text more specialized to the free trial supress, and only for those we simply `return` instead of throw (below)

---

only unauth 401 for continues free tial error will be suppressed for now, rest all 401 still shows up.